SetDemo/src/SetDemo.java

```diff
@@ -0,0 +1,25 @@
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.Set;
+
+public class SetDemo {
+    public static void main(String[] args) {
+
+        LinkedList<String> list=new LinkedList<>();
+        list.add("Petr");
+        list.add("Anna");
+        list.add("Maria");
+        list.add("Ivan");
+        list.add("Anna");
+        list.add("Petr");
+        list.add("Filip");
+        list.add("Anna");
+        list.add("George");
+
+        //System.out.println(list);
+        Set<String> set= new HashSet<>();//inner class?
+
+        set.addAll(list);// Set allows no duplicates, consequently they will be eliminated by adding them to a set
```

Comment:
This is a demo but not a solution to a task which is always a separate method with a test.